client/packages/system/src/Item/Components/PackVariant/PackVariantEntryCell.tsx

```diff
@@ -65,10 +65,16 @@ export const PackVariantEntryCell =
       return (
         <NumericTextInput
           focusOnRender={shouldFocusInput}
+          min={1}
           value={packSize}
           onChange={newValue => {
-            setPackSize(newValue || 1);
-            updater({ ...rowData, [column.key]: newValue });
+            // newValue could be undefined, while the user is inputting
+            // (e.g. they clear the field to enter a new pack size)
+            // In that case, we temporarily update the column value to 0 (so we don't have any `NaN`s)
+
+            // NumericTextInput onBlur will reset the value to our min (1) if the field is empty!
+            setPackSize(newValue);
+            updater({ ...rowData, [column.key]: newValue || 0 });
```

Comment:
Add handling of undefined pack size while someone is entering a new pack size. Should be reset to at least 1 on blur